docs/source/tutorials/otx_cli/how_to_train/detection.rst

```diff
@@ -48,36 +48,41 @@ The following example creates a virtual environment in the ``det_venv`` folder f
 Dataset preparation
 ***************************
 
-1. Download a public `BCCD dataset <https://public.roboflow.com/object-detection/bccd/3>`_ (login required). Log in, click ``Download`` button and chose ``Terminal`` option. You will get the code line like this, but with your personal API key.
+1. Clone a repository with `WGISD dataset <https://github.com/thsant/wgisd>`_. 
 
 .. code-block::
+  cd data
+  git clone https://github.com/thsant/wgisd.git
+  cd wgisd
+  git checkout 6910edc5ae3aae8c20062941b1641821f0c30127
 
-  curl -L "https://public.roboflow.com/ds/<YOUR_API_KEY>" > bccd.zip;
-  unzip bccd.zip; rm bccd.zip
 
-This dataset contains images of blood cells. It's a great example to start with, because the training model achieves high accuracy right grom the beginning due to large and focused objects.
+This dataset contains images of grapevines with the annotation for different varieties of grapes. 
+- ``CDY``	- Chardonnay
+- ``CFR``	- Cabernet Franc
+- ``CSV``	- Cabernet Sauvignon
+- ``SVB``	- Sauvignon Blanc
+- ``SYH``	- Syrah
 
-.. image:: ../../../../utils/images/bccd_sample_image.jpg
+It's a great example to start with. The model achieves high accuracy right from the beginning of the training due to large and focused objects. Also, these objects are distinguished by a person, so we can check inference results just by looking at images.
+
+.. image:: ../../../../utils/images/wgisd_dataset_sample.jpg
```

Comment:
Can we add the second image with bboxes for comparison?

Reply:
I didn't want to put 2 images here not to extend page too much. And  I wanted to use this image without annotation to run demo on.